src/Apps/Settings/Routes/Insights/Components/__tests__/InsightsHeader.jest.tsx

```diff
@@ -25,25 +24,12 @@ describe("InsightsHeader", () => {
     }))
   })
 
-  describe("In mobile view", () => {
```

Comment:
**note:** we don't need this suite anymore since the component is only going to be rendered in small screens from now on